### PR TITLE
Document how to get logs

### PR DESCRIPTION
**Pull request by chilagrow**
# Description

Closes https://github.com/FerretDB/FerretDB/issues/2354.

## Readiness checklist

* [ ] I added/updated unit tests.
* [ ] I added/updated integration/compatibility tests.
* [ ] I added/updated comments and checked rendering.
* [ ] I made spot refactorings.
* [x] I updated user documentation.
* [ ] I ran `task all`, and it passed.
* [x] I ensured that PR title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

File: website/docs/logs.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+sidebar_position: 10
+description: Logs
+---
+
+# Logs
```
Comment (chilagrow) [inline website/docs/logs.md:6]: Not sure where this can be best located.

**Comment by chilagrow** [comment]
Added `do-not-merge` tag while I check if I've missed something.

**Review by rumyantseva — APPROVED** [review]
The explanation looks clear and correct to me, I'm not sure if @AlekSi wanted to include more info though.

**Review by w84thesun — APPROVED** [review]
👍

File: website/docs/logs.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
```
Comment (Fashander) [inline website/docs/logs.md:7]:
```suggestion

# Logging

``````suggestion

# Logs

``````suggestion

# Logs

``````suggestion

# Logs

``````suggestion

# Logs

``````suggestion

# Logs

``````suggestion

# Logs

``````suggestion

# Logs

```

File: website/docs/logs.md (hunk 1 above)
Comment (Fashander) [reply website/docs/logs.md:6]:
```suggestion
# Logging
```

File: website/docs/logs.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+## Docker logs
+
+Logs from FerretDB running on Docker can be accessed through the container.
+
+If the Docker was launched with the [quick start](quickstart-guide/docker.md#setup-with-docker-compose),
```
Comment (Fashander) [inline website/docs/logs.md:12]:
```suggestion
If Docker was launched with the [quick start](quickstart-guide/docker.md#setup-with-docker-compose),
```

File: website/docs/logs.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+---
+sidebar_position: 10
+description: Logs
+---
+
+# Logs
```
Comment (Fashander) [inline website/docs/logs.md:6]:
```suggestion
# Logging
```

**Review by Fashander — DISMISSED** [review]
LGTM. I left a few some suggestions

**Review by Fashander — APPROVED** [review]

**Comment by AlekSi** [comment]
CleanShot 2023-04-05 at 08 12 45@2x

@Fashander Is the new sidebar structure good enough for you?

**Review by w84thesun — APPROVED** [review]

**Review by rumyantseva — APPROVED** [review]

**Pull request merged**